lib/full_node.js

```diff
@@ -0,0 +1,14 @@
+//      
+var bcoin = require('bcoin');
+var TrustIsRisk = require('./trust_is_risk');
+
+class FullNode extends bcoin.fullnode {
+                     
+
+  constructor(options         ) {
+    super(options);
+    this.trust = new TrustIsRisk(this);
```

Comment:
Possibly, however I thought we agreed that the initial version will be bitcoin full node. This is [in the spec](https://docs.google.com/document/d/11NkAz5-Am-XVhjOduzAX5Ao4KL7qlewgD7JzDI55Q_c/edit) and was discussed in the review meeting.

Please note that you don't have to use the full node class provided here, you can just use the TIR class alone and connect it to any other type of node manually. We can provide a spv_node implementation after the initial version is working.